File: app/src/main/java/com/tangem/data/network/EstimatefeeApi.java
package com.tangem.data.network;

import retrofit2.Call;
import retrofit2.http.GET;

public interface EstimatefeeApi {
    @GET(Server.ApiEstimatefee.Method.N_2)
    Call<String> getEstimateFeePriority();

    @GET(Server.ApiEstimatefee.Method.N_3)
    Call<String> getEstimateFeeNormal();

    @GET(Server.ApiEstimatefee.Method.N_6)
    Call<String> getEstimateFeeMinimal();
}

File: blockchain/src/main/java/com/tangem/blockchain/bitcoin/network/api/EstimateFeeApi.kt
package com.tangem.blockchain.bitcoin.network.api

import retrofit2.http.GET

interface EstimatefeeApi {
    @GET(ESTIMATE_FEE_URL + "n/2")
    suspend fun getEstimateFeePriority(): String

    @GET(ESTIMATE_FEE_URL + "n/3")
    suspend fun getEstimateFeeNormal(): String

    @GET(ESTIMATE_FEE_URL + "n/6")
    suspend fun getEstimateFeeMinimal(): String
}

const val ESTIMATE_FEE_URL = "https://estimatefee.com/"